src/components/_footer.scss

```diff
@@ -2,12 +2,12 @@
 
 /// Footer component
 .footer {
```

Comment:
We can remove the footer now can't we? Now that we've rolled out the global nav?